bigquery_etl/dryrun.py

```diff
@@ -253,6 +253,16 @@
     # Tests
     "sql/moz-fx-data-test-project/test/simple_view/view.sql",
 }
+SKIP.update(
+    [
+        p
+        for f in [Path(s) for s in SKIP]
+        for p in glob.glob(
+            f"sql/{TEST_PROJECT}/{f.parent.parent.name}*/{f.parent.name}/{f.name}",
+            recursive=True,
+        )
+    ]
+)
```

Comment:
We still want to skip dry runs even for those files that get copied to the stage project, so add their paths here